client/sync.c

```diff
@@ -246,8 +246,12 @@ void *sync_thread(void *arg)
 	for (;;) {
 		xtime_t st = xdag_get_xtimestamp();
 		if (st - t >= MAIN_CHAIN_PERIOD) {
-			t = st;
-			request_blocks(0, 1ll << 48);
+            if(g_xdag_state != XDAG_STATE_LOAD) {
```

Comment:
This will cause problem of unsynced status for pool if there is blocks missing, and no other ways to sync such block only when the block is checked later, but it means the balance of wallets may be incorrect.

Reply:
You are right, this problem does exist here, I will change it back